packages/console/src/consts/subscriptions.ts

```diff
@@ -34,3 +36,6 @@ export const featuredPlanIds: string[] = [ReservedPlanId.Free, ReservedPlanId.Pr
 export const featuredPlanIdOrder: string[] = [ReservedPlanId.Free, ReservedPlanId.Pro];
 
 export const checkoutStateQueryKey = 'checkout-state';
+
+/** The latest pro plan id we are using. TODO: Remove this when we have the new Pro202411 plan released. */
+export const latestProPlanId = isDevFeaturesEnabled ? ReservedPlanId.Pro202411 : ReservedPlanId.Pro;
```

Comment:
Is this necessary? Both the values of them are `pro`

Reply:
The value for `ReservedPlanId.Pro202411` is `pro-202411`